gapic/src/main/com/google/gapid/perfetto/models/CounterInfo.java

```diff
@@ -45,6 +46,9 @@
       "where (value + 1 > value or value - 1 < value) " +
       "group by ct.id";
 
+  private static final String LIST_COUNTER_GROUP_SQL =
+      "select group_id, track_id from gpu_counter_group";
+
```

Comment:
Why not join with the group table in the above list query and make the group_id a field of the `CounterInfo`?

Reply:
A counter can be in multiple groups.  The counter group map make it easier to add the track in the correct order.  It's either we build it here or later in Tracks.enumerateGpu.
If we are building the table here, it wouldn't make sense to join with the list_sql query, would it?

Reply:
Yeah, when I wrote this, I hadn't realized that counters can be in multiple groups.